HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/Building.cs

```diff
@@ -0,0 +1,22 @@
+﻿using GraphicalEditor.Enumerations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Models.MapObjectRelated
+{
+    class Building : MapObjectEntity
+    {
+        private int _numOfFloors;
+
+        public Building(String description, int numOfFloors)
+            : base(new MapObjectType(MapObjectTypes.BUILDING), "Building 1")
+        {
+            this.NumOfFloors = numOfFloors;
```

Comment:
Using 'this' here isn't necessary.

Reply:
Although using **this** is pretty much redundant in these cases, I really like using it since it helps me to quickly distinguish class fields from other variables, but I will avoid using it in this project if you don't like that behavior.